scout/server/config.py

```diff
@@ -68,3 +68,9 @@
 RANK_MODEL_LINK_POSTFIX = "-.ini"
 SV_RANK_MODEL_LINK_PREFIX = "https://github.com/Clinical-Genomics/reference-files/blob/master/rare-disease/rank_model/svrank_model_-v"
 SV_RANK_MODEL_LINK_POSTFIX = "-.ini"
+
+
+# MatchMaker
+MME_ACCEPTS = "application/vnd.ga4gh.matchmaker.v1.0+json"
```

Comment:
These lines need to be commented out in master branch since there is no MatchMaker connection in the default Scout. What you could do instead is add them as comments to `docker-compose-mm.yml`, so we remember how config should be modified for that specific docker-compose file to work